cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -486,5 +576,6 @@
       "type": "object"
     }
   },
-  "type": "object"
+  "type": "object",
+  "required": ["accelerator", "collections", "date_created", "date_published", "experiment", "license", "publisher", "recid", "title", "type"]
```

Comment:
Are you sure that all the records have these fields? E.g. the author list records don't have `accelator`, so we cannot require it.  

Here is a quick simple way to check:

```console
$ cd cernopendata/modules/fixtures/data/records
$ grep -c '"accelerator"' * | grep :0$
atlas-learning-resources.json:0
cms-author-list-2012.json:0
cms-author-list.json:0
cms-author-list-Run2011A.json:0
opera-author-list-multiplicity.json:0
opera-author-list-tau.json:0
```

... meaning that we cannot make it mandatory.   Or else we would have to amend the records.

(I have some more detailed JSON Schema checks, but that would require to install some Python libraries...) detailed check o...

Reply:
So, I checked everything and I had to take out `abstract.description`, `accelerator`, `date_created` and `license`. There are also 3 files that don't have the `date_published` property, but I'll add it there, because I think it should definitely be required. 

For `authors.name` and `collaboration.name`, perhaps we could keep them by using something like `Oneof`? 

Also, it would be better if we could add `license` everywhere, since it's missing in a lot of records. Would this be easy to do programmatically? It should be just adding CC0 in most, if not all, these records.

Reply:
As we discussed IRL, we can revisit the `authors` and `collaboration` properties in the future; for now, we will only keep `experiment` as required.